xml/Microsoft.Extensions.Primitives/StringValues.xml

```diff
@@ -1000,7 +1000,15 @@ In application code, this field is most commonly used to safely represent a <xre
         <param name="values">A <see cref="T:Microsoft.Extensions.Primitives.StringValues" /> to implicitly convert.</param>
         <summary>Defines an implicit conversion of a given <see cref="T:Microsoft.Extensions.Primitives.StringValues" /> to a string, with multiple values joined as a comma separated string.</summary>
         <returns>Returns <see langword="null" /> where <see cref="T:Microsoft.Extensions.Primitives.StringValues" /> has been initialized from an empty string array or is <see cref="P:Microsoft.Extensions.Primitives.StringValues.Empty" />. Otherwise, it returns the implicit conversion of <see cref="T:Microsoft.Extensions.Primitives.StringValues" /> to a string.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Returns `null` where <xref:Microsoft.Extensions.Primitives.StringValues> has been initialized from an empty string array or is <xref:Microsoft.Extensions.Primitives.StringValues.Empty>.
+
+          ]]></format>
+        </remarks>
```

Comment:
This information is redundant, since it should go in the `<returns>` section and it's already there.

```suggestion
        <remarks>To be added.</remarks>
```